- создал CustomResource и CustomResourceDefinition для mysql оператора

woworks opened the pull request:
- создал часть логики mysql оператора при помощи python KOPF
- собрал образ, запушил его в docker hub и сделал деплой оператора
- исправил ошибки в mysql-operator.py

# Выполнено ДЗ №

- [x] Основное ДЗ
- [ ] Задание со *

## В процессе сделано:
- Пункт 1
- Пункт 2

## Как запустить проект:
- Например, запустить команду X в директории Y

## Как проверить работоспособность:
- Например, перейти по ссылке http://localhost:8080

## PR checklist:
- [x] Выставлен label с темой домашнего задания

woworks committed:
- создал CustomResource и CustomResourceDefinition для mysql оператора
- создал часть логики mysql оператора при помощи python KOPF
- собрал образ, запушил его в docker hub и сделал деплой оператора
- исправил ошибки в mysql-operator.py

woworks committed:
- создал CustomResource и CustomResourceDefinition для mysql оператора
- создал часть логики mysql оператора при помощи python KOPF
- собрал образ, запушил его в docker hub и сделал деплой оператора
- исправил ошибки в mysql-operator.py

woworks committed:
- создал CustomResource и CustomResourceDefinition для mysql оператора
- создал часть логики mysql оператора при помощи python KOPF
- собрал образ, запушил его в docker hub и сделал деплой оператора
- исправил ошибки в mysql-operator.py

Konstantinov86 commented:
Необходимо  разрешить  конфликт мерджа - и тесты почему -то  падают - error: arguments in resource/name form must have a single resource and name -локально все отработало? можете приложить  к дз  требуемый из задания скриншот ?

woworks committed:
## kubernetes-monitoring
- собрал образ nginx с stub_status
- установил prometheus с помощью оператора
- создал deployment с кастомным nginx образом и сайд-контейнером nginx-prometheus-exporter
- создал service и servicemonitor

woworks commented:
странно, но пришлось поменять json_manifest = yaml.load(yaml_manifest) на json_manifest = yaml.load(yaml_manifest, Loader=yaml.Loader) - была ошибка в скрипте
![Screenshot from 2022-08-28 19-49-24](https://user-images.githubusercontent.com/2965982/187086769-12cfcb53-00e0-496d-a9d9-286275df01da.png)

Вообще, странно что питон скрипт из гиста для домашки отличается от того, который в образе wenger23/mysql-operator:1.1

а конфликт мерджа наверно потому, что я еще одну ветку запушил для следующей домашки, и git не может автоматом разрешить конфликт

Konstantinov86 commented:
нет  -у вас  конфликт с  readme файлом

woworks committed:
Merge branch 'main' into kubernetes-operators

Konstantinov86 commented:
Добрый день! Дз принято - отличная  работа
